src/app/views/censusdata/create.html

```diff
@@ -0,0 +1,171 @@
+<div class="container">
+    <div class="well row" >
+        <h4>Census participants please be as complete as possible:</h4>
+        <ul>
+            <li>Review form guidelines prior to census.</li>
+            <li>Fill in/check box for all information in top section side one.</li>
+            <li>Always provide "number of adults and juveniles" data for species.</li>
+            <li>Always observe birds from at least 50 yards away.</li>
+        </ul>
+    </div>
+                
+    <form class="form-horizontal" role="form"> 
+        <div class="form-group panel panel-primary">
+            <div class="panel-heading">Section One</div>
+            <div class="panel-body">              
+            <div class="form-group">
+                <label class="control-label col-md-3" for="colonyCode">Colony Code:</label>
+                <div class="col-md-3">
+                <input type="text" class="form-control" id="colonyCode" placeholder="Enter Colony Code">
+                </div>    
+            </div> 
+            <div class="form-group">
+                <label class="control-label col-md-3" for="colonyName">Colony Name:</label>
+                <div class="col-md-9">
+                <input type="text" class="form-control" id="colonyName" placeholder="Enter Colony Name">
+                </div>
+            </div>
+            <div class="form-group">
+                <label class="control-label col-md-3" for="surveyDate">Survey Date:</label>
+                <div class="col-md-4">
+                <input type="date" class="form-control" id="surveyDate" placeholder="Enter Survey Date">
+                </div>
+            </div>   
+            <div class="form-group">
+                <label class="control-label col-md-3" for="timeStart">Time Start:</label>
+                <div class="col-md-3">
+                <input type="time" class="form-control" id="timeStart" placeholder="Enter Survey Date">
```

Comment:
I haven't tried the HTML5 "time" control. On the workday create/edit screens, I used an Angular controller for time, having completely forgotten that HTML 5 supports this. I need to look more closely at the HTML5 "time" input type.